frontend/src/components/Home/Home.js

```diff
@@ -1,38 +1,256 @@
+                <img src={valueVisonary} alt="two people at a podium discussion"/>
 
+            </ValueWrapper>
+            <Wrapper>
+                <div>
+                    <BlueHeadline>Our way of working</BlueHeadline>
+                    <WayOfWorking>
+                        <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</li>
+                        <li>Lorem ipsum.</li>
+                        <li>Lorem ipsum dolor sit amet, consectetur.</li>
+                        <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</li>
+                    </WayOfWorking>
+                </div>
+
+                <BoxNext>
+                    <h3>Ohhh, what's next?</h3>
+                    <p>To give you an overview of upcoming tasks, we prepared a personal checklist for you.</p>
+                    <img src={checklistIconLight} alt=""/>
+                </BoxNext>
+
+                
+                    <p>If you have any doubts or questions, just let us know</p>
+                    <ContactWrapper>
+                        <a href="mailto:annika.vennemann@youth-against-aids.org">
+                            <img src={mailIcon} alt="" />
+                        </a>
+                        <a href="tel:0049-173-3975041">
+                            <img src={phoneIcon} alt="" />
+                        </a>
+                    </ContactWrapper>
+            </Wrapper>
+            <Navigation />
         </>
     )
 }
 
+const Header = styled.header`
+    margin: 0;
+    background: linear-gradient(0deg, rgba(255,255,255,1) 0%, rgba(208,208,208,1) 100%);
+`
+
+const HeaderImg = styled.img`
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    padding: 1em;
+`
+
+const Wrapper = styled.div`
+    margin: 0.5em 2em;
+    
+    p {
+        text-align: center;
+        font-size: 20px;
+    }
+`
+
 const Headline = styled.h2`
+    margin: 0 0.3em;
     font-size: 32px;
     font-weight: lighter;
     color: #F5BAC4;
 
-span{
-    font-family: Yellowtail;
-    font-size: 32px;
-    font-weight: 300;
-    color: #707070
-}  
+    span{
+        font-family: Yellowtail;
+        font-size: 32px;
+        font-weight: 300;
+        color: #707070
+    }  
 `
+
+const WelcomeBox = styled.div`
+    margin: 1em 0 0 0;
+    padding: 1.5em 2em;
+    box-shadow: 4px 4px 18px #00000029;
+    border-radius: 20px;
+    display: flex;
+    flex-direction: column;
+
+    p {
+        text-align: center;
+        font-size: 20px;
+    }
+
+    img {
+        align-self: flex-end;
+    }
+`
+
+const InfoBox = styled.div`
+    display: flex;
+    justify-content: center;
+    font-size: 0.8em;
+    color: #029FE3;
+
+    img {
+        padding-right: 0.6em;
+    }
+`
+const Vision = styled.ul`
+    list-style-image: url("../../images/bulletpoint.svg");
+
+    li {
+        font-size: 20px;
+    }
+`
+
+const ValueWrapper = styled.div`
+    margin: 0 0 0 -0.5em;
```

Comment:
same like above :) makes it more explicit
```suggestion
    margin-right: -0.5em;
```